docs/analog/commands/DataSegment.md

```diff
@@ -65,13 +65,13 @@ Denotes the index or position of a particular segment within the entire message.
 
 ### Examples
 
-| Field                   | Value                                                                  | Bits         | Hex                |
-| ----------------------- | ---------------------------------------------------------------------- | ------------ | ------------------ |
-| command id              | `30`                                                                   |              | `0x1e`             |
-| command size            | `9`                                                                    |              | `0x09`             |
-| segmentation session id | `35`                                                                   |              | `0x23`             |
-| segment flags           | last segment: `true` <br> segments number: `1` <br> segment index: `1` | `0b10010001` | `0x91`             |
-| data                    | `23 10 10 07 00 00 00`                                                 |              | `0x23101007000000` |
+| Field                   | Value                                                                    | Bits         | Hex                |
+|-------------------------|--------------------------------------------------------------------------|--------------|--------------------|
```

Comment:
all tables used spaces between columns, like this

```
| Field                   | Value                                                                  | Bits         | Hex                |
| ----------------------- | ---------------------------------------------------------------------- | ------------ | ------------------ |
| command id              | `30`                                                                   |              | `0x1e`             |
```

use autoformatting from https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one extension